integration-tests/steps.go

```diff
@@ -233,4 +233,46 @@ var happyPathSteps = []Step{
 			},
 		},
 	},
+	{
+		action: UnbondTokensAction{
+			chain:      0,
+			unbondFrom: 0,
+			sender:     0,
+			amount:     11111111,
+		},
+		state: State{
+			0: ChainState{
+				ValPowers: &map[uint]uint{
+					0: 500,
+					1: 500,
+					2: 500,
+				},
+			},
+			1: ChainState{
+				ValPowers: &map[uint]uint{
+					// Voting power on consumer should not be affected yet
+					0: 511,
+					1: 500,
+					2: 500,
+				},
+			},
+		},
+	},
+	{
+		action: RelayPacketsAction{
+			chain:   0,
+			port:    "provider",
+			channel: 0,
+		},
+		state: State{
+			1: ChainState{
+				ValPowers: &map[uint]uint{
+					0: 500,
+					1: 500,
+					2: 500,
+				},
+			},
+		},
+	},
+	// TODO: Test full unbonding functionality, considering liquidity after unbonding period, etc.
```

Comment:
Shouldn't this be part of this integration test?

Reply:
Yes, definitely. I'm having trouble manipulating the chains in such a way that would allow this (more extended) test to function tho, so I made it a TODO for now. Essentially, I'm setting custom genesis parameters that don't seem to be registered by the chains that are spawned up in the integration test docker container. I will be looking more into it today